tests/testthat/test-build-articles.R

```diff
@@ -45,7 +45,7 @@ test_that("articles don't include header-attrs.js script", {
   pkg <- as_pkgdown(test_path("assets/articles"))
   withr::defer(clean_site(pkg, quiet = TRUE))
 
-  expect_snapshot(path <- build_article("standard", pkg))
+  suppressMessages(path <- build_article("standard", pkg))
```

Comment:
I wish there could be an option set with withr, so that the code wouldn't be nested inside `supressMessages()`.

Reply:
We might get there one day, but I think that's still quite a lot of refactoring away.